tests/Yaml/Form/Fixture/my_form.form.yaml

```diff
@@ -29,3 +29,39 @@ finishers:
       format: html
       attachUploads: true
     identifier: EmailToSender
+-----
+finishers:
+  -
+    options:
+      recipients:
+        foo@domain.com: 'Stadtwerke Bonn'
+        bar@domain.com: Bar
+      subject: 'Kontaktanfrage von Website'
```

Comment:
Btw, I didn't update these manually. All you need to do is run the test case with UT=1:

```bash
UT=1 vendor/bin/phpunit tests/SomeRectorTest.php
```

And Rector will update the 2nd half of test fixtures for you :)

Reply:
Didn´t know that